hack/00-osd-managed-cluster-config-integration.yaml.tmpl

```diff
@@ -7135,6 +7135,19 @@ objects:
     resourceApplyMode: Sync
     applyBehavior: CreateOnly
```

Comment:
Because it's in this folder, it will only be created, not updated. I don't think that's our intention. We probably want to make a subfolder?